elvish-mode.el

```diff
@@ -93,8 +104,80 @@ expected.")
       (optional "." (optional (one-or-more digit))))
   "The regex to identify elvish numbers")
 
+(defcustom elvish-builtin-functions
+  '(
+    ;; Trivial builtin
+    "nop"
+    ;; Introspection
+    "kind-of" "is" "eq"
+    ;; Value output
+    "put"
+    ;; Bytes output
+    "print" "echo" "pprint" "repr"
+    ;; Bytes to value
+    "slurp" "from-lines" "from-json"
+    ;; Value to bytes
+    "to-lines" "to-json"
+    ;; Exception and control
+    "fail" "multi-error" "return" "break" "continue"
```

Comment:
`return`, `break`, and `continue` are already being highlighted as keywords. I know they are implemented as functions, but I prefer the look of the keyword-face for them. Maybe just add `fail` and `multi-error` to that list instead?